src/utils/iqeEnablement.ts

```diff
@@ -16,6 +16,7 @@ const AUTH_ALLOWED_ORIGINS = [
   location.origin,
   /https:\/\/api(?:\.[a-z]+)?\.openshift(?:[a-z]+)?\.com/,
   /https:\/\/api?\.demo-experience(?:\.[a-z]+)?\.demo?\.redhat?\.com/,
+  /https:\/\/api?\.aws?\.ap-southeast-1(?:\.[a-z]+)?\.openshift?\.com/,
```

Comment:
Just a note. We should start thinking about OCP env variables and pass this list down from chrome.

Reply:
@aneelac22 There will be many new regional endpoints following the pattern api.<PROVIDER>.<REGION>.<ENV>.openshift.com. Can we make this more generic so that we don't have to request a change for each new region?